renovate.json

```diff
@@ -1,44 +1,46 @@
 {
-  "extends": [
-    "config:base",
-    ":disableMajorUpdates"
-  ],
-  "labels": [
-    "area / devOps",
-    "priority / low",
-    "status / code review",
-    "type / technical debt"
-  ],
-  "reviewers": [],
-  "ignoreDeps": [
-    "square/connect",
-    "spatie/laravel-sluggable"
-  ],
-  "prConcurrentLimit": 1,
-  "prHourlyLimit": 0,
-  "rebaseStalePrs": true,
-  "vulnerabilityAlerts": {
+    "extends": [
+        "config:base",
+        ":disableMajorUpdates"
+    ],
     "labels": [
-      "area / devOps",
-      "priority / high",
-      "status / code review",
-      "type / technical debt"
-    ]
-  },
-  "hostRules": [
-    {
-      "domainName": "nova.laravel.com",
-      "username": "developers@robojackets.org",
-      "encrypted": {
-        "password": "m4fVZTnuxkpRyuA3NoJZg2tXRawuBdt7IW21BXPWWaB7tzPxd2ck8U/9FHr3omnrnRAdCtvUILRxGcHlBN8uhQlxRyhfxchSgtlvDN5bVzBmzWp5VbKjWpuUxq2mU7ZqrYJJBtLP7SlwztOhZbRKjP5IscNgUWnbHGOPdhEsfbnUkfFXKm7eiQlzObnEvG/CN9nGn2auROi7q/xPqJ8aCKL/NeF6ZfLlYbjREjZaC6xwOaZeWIJtD6GQ8VRN9B0qeScsHP8hTzbRZ5Fe5CKL82yY5N998a7O8SXVNYNhIwKzrYHko5Vg/CpbC4ZXvXiRR33pNC36W4jqMA9wXo1rZw=="
-      }
+        "area / devOps",
+        "priority / low",
+        "status / code review",
+        "type / technical debt"
+    ],
+    "reviewers": [],
+    "ignoreDeps": [
+        "square/connect",
+        "spatie/laravel-sluggable",
+        "beyondcode/laravel-dump-server",
+        "oitnetworkservices/buzzapiclient"
```

Comment:
This is available in our repository now, yes? Renovate should be able to pull it then.